src/main/java/com/uber/cadence/testing/WorkflowShadower.java

```diff
@@ -34,6 +34,7 @@
 
 public final class WorkflowShadower {
   private static final long SLEEP_INTERVAL = 300L;
+  private static final int PAGE_SIZE = 1000;
```

Comment:
I think server has a default value to this?

Reply:
removed